.gitignore

```diff
@@ -15,3 +15,4 @@ benchmark/*
 .Rprofile
 /doc/
 /Meta/
+docs
```

Comment:
Any reason for this? Is Roxygen generating to this directory?

Reply:
pkgdown is. So if you are testing site locally, you don't want to check in the results, as the site is deployed via gh pages and gh actions!